code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -621,6 +654,7 @@
 /datum/tarot/reversed/the_magician
 	name = "I - The Magician?"
 	desc = "May no harm come to you."
+	extended_desc = "grants the user the power of a powerful mage upon use, throwing everything away from the user."
```

Comment:
the `power of a powerful mage` sounds a bit too repetitive, try a synonym here I think

Reply:
reworded it to not be repetitive